### ISSUE TITLE
Pop!_OS package overview and search

### ISSUE THREAD

**Issue by PanderMusubi** [pop-os/website#19]
Please offer a Pop!_OS package overview search similar to https://packages.ubuntu.com/ or https://www.debian.org/distrib/packages

**Comment by doomspork** [comment]
Howdy @PanderMusubi :wave: Thanks for the great suggestion! This is something we'd like to make available and it's already on our roadmap :tada:  I don't have a timeline just yet but we've begun the internal conversations on how best to move forward.

Thanks for the feedback and have a great week :tada:

**Comment by rawleyfowler** [comment]
Is there any update on this?

**Comment by thatLeaflet** [comment]
I agree that this is a great idea. Many times I consider distrohopping to Pop!_OS, but I first want to check if it has the packages I want as well as the versions I want.

But in the current state, it's a bit tedious that I would need to first boot into an ISO, run sudo apt update, then search with apt to find what I'm looking for.

**Comment by baryluk** [comment]
I would also like this available. I develop some semi-complex scripts for Debian based distros, and often need to know which packages are available out of the box on each distro and distro version. I.e. versioned versions of gcc and llvm, and few more.

**Comment by PanderMusubi** [comment]
It will also help https://distrowatch.com/

An additional requirement is to perhaps also show searsch results from flathub, or at least link to searching in flathub.

**Comment by PanderMusubi** [comment]
Hope to see this available with the next LTS release in 2024.

**Comment by PanderMusubi** [comment]
With the release of COSMIC on the horizon, can this issue get picked up?

**Comment by leviport** [comment]
Still no timeline, sorry.

**Comment by jacobgkau** [comment]
For the record, I think this was closed because https://apt.pop-os.org/poparazzi/ now lists the packages & versions in the first-party Pop!_OS release repo (as well as the System76 Ubuntu PPA).
